### ISSUE TITLE
Allow action button with unmatched condition remain display/function.

### ISSUE THREAD

**Issue by ououn** [JannisX11/blockbench#2488]
### Detailed description of your suggestion

Currently, when an action button don't met it's condition, it will disappear. When multiple action buttons appear/disappear follow by condition changes, their appearance/disappearance from the place where they got assigned to causing ui size(usually height) changes.

Also allow action with unmatched condition run another set of code, which allow plugin creator, for example pop a showMessageBox to tell user in what condition this action can be use.

TLDR:
1. Option for plugin creator, that allow action with unmatched condition remain display but icon changed to darker tone, and
2. When pressed, allow action with unmatched condition run separate code.
